README.md

```diff
@@ -32,30 +34,30 @@ form. You can:
 Support on Ko-Fi allows me to develop such tools as this one and perform
 hardware-related experiments.
 
-## How Does It Look Like?
+## What Does It Look Like?
 
-Title page
+### Title Page
 
 ![Preview 1](https://user-images.githubusercontent.com/1590880/93708766-32ab0d80-fb39-11ea-8365-da2ca1b13d8b.jpg)
 
-Property filter
+### Property Filter
 
 ![Preview 2](https://user-images.githubusercontent.com/1590880/93708599-e01d2180-fb37-11ea-96b6-5d5eb4e0f285.jpg)
 
-Component detail
+### Component Detail
 
 ![Preview 3](https://user-images.githubusercontent.com/1590880/93708601-e0b5b800-fb37-11ea-84ed-6ba73f07911d.jpg)
 
 
 ## How Does It Work?
 
-The page has no backend so it can be easily hosted on GitHub Pages. Therefore,
-Travis CI download XLS spreadsheet from the JLC PCB page, a Python script
-process it and it generates per-category JSON file with components.
+The page has no backend so it can be easily hosted on GitHub Pages.
+Travis CI downloads the XLSX spreadsheet from the JLCPCB page, a Python script
```

Comment:
When we're updating the docs, it is worth it to change Travis to GH Action we use.

Reply:
I agree. That's why I made the change. This is the only instance of Travis.

Reply:
I probably explained myself wrongly; the original wording is out-dated. We no longer use Travis CI, we use GH Actions. However, even after your update, you say we use Travis.

Reply:
Good catch, thank you. Fixed.